src/App.tsx

```diff
@@ -16,35 +18,94 @@
 ];
 
 export const App: React.FC = () => {
+  let goods = [...goodsFromServer];
+  let isReverseCLicked = false;
+  const ALPHABETICALLY: string = 'alphabetically';
+  const BY_LENGTH: string = 'by length';
+  const RESET: string = 'reset';
+  const [sortedGoods, setSortedGoods] = useState('');
+  const [reverse, setReversed] = useState(false);
+
+  function sortGoods() {
+    if (sortedGoods === ALPHABETICALLY) {
+      goods = goods.toSorted((goodA, goodB) => goodA.localeCompare(goodB));
```

Comment:
[BUG] - It seems like you wanted to use the `sort` method, but you wrote `toSorted`. There's no such method in JavaScript. The correct method is `sort`. The same applies to the `toReversed` method on line 39, it should be `reverse`.